test/utils.spec.js

```diff
@@ -609,6 +609,37 @@ describe('utils', () => {
       }).to.throw();
     });
   });
+
+  describe('uuid', () => {
+    describe('crypto.randomUUID', () => {
+      const crypto = require('crypto');
+      let randomUUID$;
+      let called;
+      beforeEach(() => {
+        called = false;
+        randomUUID$ = crypto.randomUUID;
+        crypto.randomUUID = () => {
```

Comment:
We might want to only run this if the function exists otherwise. As it stands, old nodes will call this rather than running the fallback code. But it's great otherwise!